ast/listener.go

```diff
@@ -271,6 +271,10 @@ func (l *listener) EnterRunStmt(c *parser.RunStmtContext) {
 	l.command.Name = "RUN"
 }
 
+func (l *listener) EnterTestStmt(c *parser.TestStmtContext) {
+	l.command.Name = "TEST"
```

Comment:
AHAH! here's where the node names are set. This code doesn't appear to be generated